platform-sdk/swirlds-common/src/main/java/com/swirlds/common/config/singleton/ConfigurationHolder.java

```diff
@@ -51,15 +51,14 @@ private ConfigurationHolder() {
     }
 
     public void reset() {
-        this.configuration = ConfigUtils.addAllConfigDataOnClasspath(ConfigurationBuilder.create())
+        this.configuration = ConfigUtils.scanAndRegisterAllConfigTypes(ConfigurationBuilder.create())
                 .build();
     }
 
     /**
      * Sets the config. This method should only be called in the browser at startup or at unit tests
      *
-     * @param configuration
-     * 		the new configuration
+     * @param configuration the new configuration
```

Comment:
I'd really appreciate, if you would not mix code changes and pure format changes.